qfieldsync/gui/map_layer_config_widget.py

```diff
@@ -91,7 +101,7 @@ def __init__(self, layer, canvas, parent):
         if layer.type() == QgsMapLayer.VectorLayer:
             self.layout().insertRow(
                 -1,
-                self.tr("Relationship configuration"),
+                self.tr("Relationship\nconfiguration"),
```

Comment:
Why don't you `QLabel.setWordWrap(true);`, instead of forcing newline? Or QT is not smart enough and makes the first column too wide?

Reply:
Not smart enough